src/domain/program.ts

```diff
@@ -211,25 +226,34 @@ export class ProgramManager
     if (code.type === 'text') {
       return {
         entrypoint: 'main:app',
-        file: new Blob([code.text], { type: 'text/plain' }),
+        file: new Blob([code.text], { type: 'text/plain' }) as File,
```

Comment:
Why not use the [File constructor](https://developer.mozilla.org/en-US/docs/Web/API/File) instead?

Reply:
I didn't write this lines so I wasn't sure if it would work changing it to a File instance. Just wanted to bypass a typing error not related with the stuff I added without breaking anything.